experiment/simulation/css/main.css

```diff
@@ -1 +1,363 @@
-/* You CSS goes in here */
\ No newline at end of file
+.window { z-index:20; }
+.jtk-connector { z-index:4; }
+.jtk-endpoint { z-index:5; }
+.jtk-overlay { z-index:6; }
+
+.columns {
+ display: flex;
+}
+
+.drag-drop {
+  display: inline-block;
+  position: absolute;
+  top: 0px;
+  width: 50px;
+  height: 50px;
+  border-radius: 0.75em;
+  border: 0.2rem solid rgb(98, 178, 235);
+  transform: translate(0px, 0px);
+  z-index: 1;
+  background-size: 100%;
+}
+
+.logic-gate{
+  background-color: #29e;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.AND {
+  background-image: url("../images/AND.svg");
+}
+
+.NOT {
+  background-image: url("../images/NOT.svg");
+}
+
+.OR {
+  background-image: url("../images/OR.svg");
+}
+
+.XOR {
+  background-image: url("../images/XOR.svg");
+}
+
+.XNOR {
+  background-image: url("../images/XNOR.svg");
+}
+
+.NAND {
+  background-image: url("../images/NAND.svg");
+}
+
+.NOR {
+  background-image: url("../images/NOR.svg");
+}
+
+.FullAdder {
+  background-color: #29e;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.FullAdder:after {
+  content: "1-bit Full Adder";
+  margin: auto;
+  padding: 1.2rem 0rem;
+  font-weight: bold;
+  font-size: 1rem;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  text-align: center;
+}
+
+.HIGH {

```

Comment:
Should HIGH and LOW be fully capitalized?

Reply:
Not necessary, I have made all class names lowercase now